requirements/base.in

```diff
@@ -6,4 +6,3 @@ PyYAML
 redo
```

Comment:
Nit: `arrow`, `PyYAML` and `taskcluster` come with `scriptworker`. I think they are redundant here, unless scriptworker removes them at some point.

Reply:
Will fix this. Now! :)